homeassistant/components/yolink/binary_sensor.py

```diff
@@ -95,6 +100,18 @@ class YoLinkBinarySensorEntityDescription(BinarySensorEntityDescription):
             device.device_type == ATTR_DEVICE_WATER_METER_CONTROLLER
         ),
     ),
+    YoLinkBinarySensorEntityDescription(
+        key="water_running",
+        translation_key="water_running",
+        device_class=BinarySensorDeviceClass.RUNNING,
```

Comment:
running is not a proper device class here

Reply:
Should I use icons.json to customize entity state icons?